node_modules/oae-content/lib/api.js

```diff
@@ -482,60 +483,73 @@ var _createContent = function(ctx, contentId, revisionId, resourceSubType, displ
-                    additionalMembers[ctx.user().id] = ContentConstants.roles.MANAGER;
-                    _setContentPermissions(ctx, contentObj, additionalMembers, function(err, contentObj) {
+                    ContentDAO.Content.createContent(contentId, revisionId, ctx.user().id, resourceSubType, displayName, description, visibility, otherValues, revisionData, function(err, contentObj, revision) {
                         if (err) {
                             return callback(err);
                         }
 
-                        var allMembers = _.union(principals, [ctx.user()]);
-                        ContentMembersLibrary.insert(contentObj, allMembers, function(err) {
+                        // Make the current user a manager of the content
+                        permissions[ctx.user().id] = ContentConstants.roles.MANAGER;
+                        _setContentPermissions(ctx, contentObj, permissions, function(err, contentObj) {
                             if (err) {
-                                log().warn({
-                                    'err': err,
-                                    'contentId': contentObj.id,
-                                    'principalIds': _.pluck(allMembers, 'id')
-                                }, 'An error occurred while inserting members into a content members library while creating a content item');
+                                return callback(err);
                             }
 
-                            // Add the content item to the specified folders, if any
-                            _addContentItemToFolders(ctx, contentObj, folders, function(err) {
+                            var allMembers = _.union(principals, [ctx.user()]);
+                            ContentMembersLibrary.insert(contentObj, allMembers, function(err) {
                                 if (err) {
-                                    log().warn({'err': err, 'contentId': contentObj.id, 'folders': folders}, 'Could not add a content item to a folder');
+                                    log().warn({
+                                        'err': err,
+                                        'contentId': contentObj.id,
+                                        'principalIds': _.pluck(allMembers, 'id')
+                                    }, 'An error occurred while inserting members into a content members library while creating a content item');
                                 }
 
-                                return callback(null, contentObj, revision);
+                                // Add the content item to the specified folders, if any
+                                _addContentItemToFolders(ctx, contentObj, folders, function(err) {
+                                    if (err) {
+                                        log().warn({'err': err, 'contentId': contentObj.id, 'folders': folders}, 'Could not add a content item to a folder');
```

Comment:
I'm not sure tbh. At this point the piece of content  is created and possible shared with a set of users. In the past, if sharing/folders failed we didn't return an error, but silenty logged a warning